src/saleor_app/workers/base.py

```diff
@@ -0,0 +1,46 @@
+import asyncio
+import logging
+from signal import SIGINT, SIGTERM, signal
+
+from saleor_app.app import SaleorApp
+from saleor_app.schemas.webhook import Webhook
+
+logger = logging.getLogger(__name__)
+
+
+class SignalHandler:
+    def __init__(self):
+        self.received_signal = False
+        signal(SIGINT, self._signal_handler)
+        signal(SIGTERM, self._signal_handler)
+
+    def _signal_handler(self, signal, frame):
+        self.received_signal = True
+
+
+class SaleorAppWorker:
+    def __init__(self, app: SaleorApp):
+        self.app = app
+        self.handlers = {
+            name: handler for name, handler in self.app.sqs_webhook_handlers if handler
```

Comment:
I am not sure that this is correct that Base class does some assigments for child class

Reply:
I'm going to ignore everything about the sqs handler for now and remove it